src/data/multiset/antidiagonal.lean

```diff
@@ -83,7 +83,7 @@ end
 @[simp] theorem card_antidiagonal (s : multiset α) :
   card (antidiagonal s) = 2 ^ card s :=
 by have := card_powerset s;
-   rwa [← antidiagonal_map_fst, card_map] at this
+  rwa [← antidiagonal_map_fst, card_map] at this
```

Comment:
This one might have been intentional, it was to line up the `have` with the `rwa`. Whether your version or the original is closed to mathlib style is probably up for debate.